content/specific-skill-success-criteria/classes-and-objects/_index.md

```diff
@@ -23,3 +23,122 @@ All students need to provably understand all of the following concepts:
 - getters and setters
 - inheritance and overriding and extending methods
 - composition
+
+# Python
+
+## Constructors
+The `__init__()` method is called the constructor and is always called when an object is created, also known as instantiating an object.
+
+### Default Constructor
+The default constructor is a simple constructor which doesn’t accept any arguments. Its definition has only one argument which is a reference to the instance being constructed.
+```
+class BankAccount:
+  def __init__(self):
+    pass
+```
+
+### Parameterized Constructor
+As the name suggests, a parameterized constructor is a constructor with parameters. Takes its first argument as a reference to the instance being constructed, known as `self` and the rest of the arguments are provided by the programmer.
+```
+class BankAccount:
+  def __init__(self, acc_no, amount):
+    self.acc_no = acc_no
```

Comment:
it would be good to use full words for variable names. We generally tell the learners to do that.